### ISSUE TITLE
HTML isn't stripped from titles when updating alt text in tab list

### ISSUE THREAD

**Issue by Rohaq** [babyman/quick-tabs-chrome-extension#187]
So I found that HTML tags added by Quicktabs based on the search term entered appear in the mouseover text.

![image](https://user-images.githubusercontent.com/246865/29101767-6222684c-7cac-11e7-827f-3c06e9e5a82b.png)

This should be easily solvable by calling the existing jQuery .text() function on the string when updating the alt text for the line.

**Comment by Rohaq** [comment]
Added a pull request that fixes this: #188
